Dockerfile

```diff
@@ -1,13 +1,21 @@
 FROM openjdk:11-slim
 MAINTAINER HMPPS Digital Studio <info@digital.justice.gov.uk>
 
-RUN apt-get update && apt-get install -y curl
-WORKDIR /app
-
-COPY build/libs/custody-api*.jar /app/app.jar
-COPY run.sh /app
+RUN apt-get update && \
+    apt-get install -y curl && \
+    rm -rf /var/lib/apt/lists/*
 
 ENV TZ=Europe/London
 RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone
 
+RUN addgroup --gid 2000 --system appgroup && \
+    adduser --uid 2000 --system appuser --gid 2000
+
+WORKDIR /app
+
+COPY --chown=appuser:appgroup build/libs/custody-api*.jar /app/app.jar
+COPY --chown=appuser:appgroup run.sh /app
+
+USER 2000
```

Comment:
Could use the newly created username rather than uid? Just thinking it might be clearer, but a small thing.

Reply:
Agreed, will change on next iteration.  Copied verbatim from a different project I'm afraid.